src/jetzig/Environment.zig

```diff
@@ -95,8 +95,7 @@ pub fn getServerOptions(self: Environment) !jetzig.http.Server.ServerOptions {
         std.process.exit(1);
     }
 
-    // TODO: Generate nonce per session - do research to confirm correct best practice.
-    const secret_len = jetzig.http.Session.Cipher.key_length + jetzig.http.Session.Cipher.nonce_length;
+    const secret_len = jetzig.http.Session.Cipher.key_length;
     const secret = (try self.getSecret(&logger, secret_len, environment))[0..secret_len];
```

Comment:
The slice could now even be omitted, couldn't it? Unless it's a feature that too long values are allowed.

Reply:
Yep, the secret key generator in the CLI tool over-allocates to allow for changes in backend crypto implemntation (such as this one).